reporters_db/data/laws.json

```diff
@@ -475,6 +475,23 @@
             "variations": []
         }
     ],
+    "C.E.T.S.": [
+        {
+            "cite_type": "treaty",
+            "end": null,
+            "examples": [
+                "C.E.T.S. No. 196"
```

Comment:
I think I would add and example that has No. and Number since we make that option available in the regex.

Reply:
Sure. No problem.